src/translations/de.ts

```diff
@@ -0,0 +1,20 @@
+import { registerTranslation } from '../utilities/localize';
+import type { Translation } from '../utilities/localize';
+
+const translation: Translation = {
+  $code: 'de',
+  $name: 'Deutsch',
+  $dir: 'ltr',
+
+  close: "Schlie\xDFen",
+  copy: 'Kopieren',
+  progress: 'Fortschritt',
+  scroll_to_end: 'Zum Ende scrollen',
+  scroll_to_start: 'Zum Anfang scrollen',
+  select_a_color_from_the_screen: "W\xE4hle eine Farbe vom Bildschirm",
```

Comment:
... and here the `ä`.

Maybe this should be `&auml;`?

Reply:
Why not just use `ä`? This should work when utf-8 is used, no?

Reply:
I think so. But I'm not sure if this also works for projects with other encodings. We ourselves still have pages with CP1252 ...
With the code points we would be on the safe side.

But: I am happy to be convinced of the opposite

Reply:
Thanks for this! All terms should be in Unicode (i.e. no escape chars or entities).

Reply:
```suggestion
  select_a_color_from_the_screen: "Wähle eine Farbe vom Bildschirm",
```